projects/plugins/automattic-for-agencies-client/src/js/components/connection-card/index.jsx

```diff
@@ -1,61 +1,333 @@
+import { getRedirectUrl } from '@automattic/jetpack-components';
+import { ConnectButton } from '@automattic/jetpack-connection';
+import { createInterpolateElement } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
-import React from 'react';
+import classNames from 'classnames';
+import React, { useCallback, useState } from 'react';
 import BrandedCard from '../branded-card';
 import CheckIcon from '../check-icon';
-import ConnectionButton from '../connection-button';
+import CloseIcon from '../close-icon';
+import LeftArrow from '../left-arrow';
 import styles from './styles.module.scss';
 
 /**
- * Connection Card component.
+ * Site Connection Content
+ * The main copy for the connection card.
  *
- * @returns {React.Component} The `ConnectionCard` component.
+ * @param {object}   props                             - Component props
+ * @param {Function} props.onShowSharingDetailsClick  - Callback to show the sharing details.
+ *
+ * @returns {React.Component} The `ConnectionContent` component.
  */
-export default function ConnectionCard() {
+function ConnectionContent( { onShowSharingDetailsClick } ) {
 	const { apiNonce, apiRoot, registrationNonce } = window.automatticForAgenciesClientInitialState;
 	return (
-		<BrandedCard>
-			<div className={ styles.card }>
-				<h1 className={ styles.card__heading }>
+		<>
+			<h1>
+				{ __(
+					'Add this site to Automattic for Agencies by connecting now',
+					'automattic-for-agencies-client'
+				) }
+			</h1>
+			<ul className={ styles.checklist }>
+				<li>
+					<CheckIcon />
+					{ __( 'See your site in the Sites dashboard', 'automattic-for-agencies-client' ) }
+				</li>
+				<li>
+					<CheckIcon />
 					{ __(
-						'Add this site to Automattic for Agencies by connecting now',
+						'View any security and performance issues across all of your sites',
 						'automattic-for-agencies-client'
 					) }
-				</h1>
-				<ul className={ styles[ 'check-list' ] }>
+				</li>
+				<li>
+					<CheckIcon />
+					{ __(
+						'Update plugins across all sites in a couple of clicks',
+						'automattic-for-agencies-client'
+					) }
+				</li>
+				<li>
+					<CheckIcon />
+					{ __( 'Receive instant downtime alerts', 'automattic-for-agencies-client' ) }
+				</li>
+				<li>
+					<CheckIcon />
+					{ __( 'And more', 'automattic-for-agencies-client' ) }
+				</li>
+			</ul>
+			<div className={ styles[ 'site-connection' ] }>
+				<div className={ styles[ 'terms-of-service' ] }>
+					{ createInterpolateElement(
+						__(
+							'By clicking the <strong>connect this site</strong> button, you agree to our <tosLink>Terms of Service</tosLink> and to <shareDetailsLink>share details</shareDetailsLink> with WordPress.com.',
+							'automattic-for-agencies-client'
+						),
+						{
+							strong: <strong />,
+							tosLink: (
+								<a
```

Comment:
Would it be an option to use `ExternalLink` here, to visually indicate that the link opens in a new window?

Reply:
Would you like to see this applied to the connection UIs of other Jetpack plugins? If so, it would make sense for consistency.

Reply:
Yeah, I think it would make sense everyone. It would be interesting to have Design's take on this, because I think the external link icons is something we were aiming for for all such links in our UIs.